src/fmu/dataio/hook_implementations/jobs.py

```diff
@@ -1,8 +1,10 @@
-from ert_shared.plugins.plugin_manager import hook_implementation
-from ert_shared.plugins.plugin_response import plugin_response
+try:
+    import ert_shared
+except ModuleNotFoundError:
+    import ert.shared as ert_shared
 
 
-@hook_implementation
-@plugin_response(plugin_name="fmu_dataio")
+@ert_shared.plugins.plugin_manager.hook_implementation
+@ert_shared.plugins.plugin_manager.plugin_response(plugin_name="fmu_dataio")
```

Comment:
For consistencies sake, you could do the same style import here as in `create_case_metadata.py`. 
```suggestion
@hook_implementation
@plugin_response(plugin_name="fmu_dataio")
```